Unable to resolve dependency

When installing through npm throws dependency error:

After running

` npm i @rainbow-me/rainbowkit wagmi ethers
`

Then
npm i @celo/rainbowkit-celo

Throws:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: rainbow-celo@0.1.0
npm ERR! Found: wagmi@0.9.6
npm ERR! node_modules/wagmi
npm ERR!   wagmi@"^0.9.6" from the root project
npm ERR!   peer wagmi@"0.9.x" from @rainbow-me/rainbowkit@0.8.1
npm ERR!   node_modules/@rainbow-me/rainbowkit
npm ERR!     @rainbow-me/rainbowkit@"^0.8.1" from the root project
npm ERR!     peer @rainbow-me/rainbowkit@">=0.8.1" from @celo/rainbowkit-celo@0.8.0
npm ERR!     node_modules/@celo/rainbowkit-celo
npm ERR!       @celo/rainbowkit-celo@"*" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer wagmi@">0.9.x" from @celo/rainbowkit-celo@0.8.0
npm ERR! node_modules/@celo/rainbowkit-celo
npm ERR!   @celo/rainbowkit-celo@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /home/basti/.npm/_logs/2023-01-26T15_04_51_765Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/basti/.npm/_logs/2023-01-26T15_04_51_765Z-debug-0.log
```

--- a/apps/example/package.json
+++ b/apps/example/package.json
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@celo/rainbowkit-celo": "0.8.0",
+    "@celo/rainbowkit-celo": "0.8.1-beta.1",
     "@mdx-js/loader": "^2.1.3",
     "@mdx-js/react": "^2.1.3",
     "@next/mdx": "^12.3.1",

--- a/apps/example/package.json
+++ b/apps/example/package.json
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@celo/rainbowkit-celo": "0.8.1-beta.1",
+    "@celo/rainbowkit-celo": "0.8.1-beta.3",
     "@mdx-js/loader": "^2.1.3",
     "@mdx-js/react": "^2.1.3",
     "@next/mdx": "^12.3.1",

--- a/apps/example/pages/index.tsx
+++ b/apps/example/pages/index.tsx
@@ -70,8 +70,9 @@ const Home: NextPage = () => {
         <h2 className={styles.title}>Celo 🌈 RainbowKit</h2>
         <ConnectButton />
         <Demo />
+        <em>Follow the instructions for <a className={styles.inlineLink} href="https://www.rainbowkit.com/docs/installation">installing rainbowkit</a> and...</em>
         <h3>Install As a Package</h3>
-        <code className={styles.install}>yarn add @celo/rainbowkit-celo @rainbow-me/rainbowkit wagmi ethers</code>
+        <code className={styles.install}>yarn add @celo/rainbowkit-celo</code>
         <h3>Or copy from source files</h3>
         <a target="_blank" rel="noopener noreferrer" className={styles.link}
             href="https://github.com/celo-org/rainbowkit-celo/tree/main/packages/rainbowkit-celo/chains">

--- a/apps/example/styles/Home.module.css
+++ b/apps/example/styles/Home.module.css
@@ -37,10 +37,9 @@
   width: 45rem;
   max-width: 90vw;
   overflow: auto;
-height: 400px;
-
+  height: 400px;
   max-height: 70vh;
-  margin: 0 5px 10px 5px;
+  margin: 15px 5px 20px 5px;
 
 }
 
@@ -50,6 +49,11 @@ height: 400px;
   margin: 4px;
 }
 
+.inlineLink {
+  text-decoration: underline;
+}
+
+.inlineLink:hover,
 .link:hover {
   background-color: lightpink;
 }

--- a/packages/rainbowkit-celo/chains/celo.ts
+++ b/packages/rainbowkit-celo/chains/celo.ts
@@ -13,6 +13,12 @@ const Celo: Chain = {
   },
   rpcUrls: {
     default: {http: ['https://forno.celo.org']},
+    infura: {
+      http: ["https://celo-mainnet.infura.io/v3/"]
+    },
+    public: {
+      http: ['https://forno.celo.org']
+    },
   },
   blockExplorers: {
     default: { name: 'Celo Explorer', url: 'https://explorer.celo.org/mainnet' },

--- a/packages/rainbowkit-celo/chains/index.ts
+++ b/packages/rainbowkit-celo/chains/index.ts
@@ -1,3 +1,3 @@
-export { default as Alfajores } from './alfajores';
-export { default as Celo } from './celo';
-export { default as Baklava } from './baklava';
+export { default as Alfajores } from './alfajores.js';
+export { default as Celo } from './celo.js';
+export { default as Baklava } from './baklava.js';

--- a/packages/rainbowkit-celo/index.ts
+++ b/packages/rainbowkit-celo/index.ts
@@ -1,2 +1,2 @@
-export * as CeloWallets from "./wallets";
-export * as CeloChains from "./chains";
+export * as CeloWallets from "./wallets/index.js";
+export * as CeloChains from "./chains/index.js";

--- a/packages/rainbowkit-celo/lists/index.ts
+++ b/packages/rainbowkit-celo/lists/index.ts
@@ -1,7 +1,7 @@
 import { connectorsForWallets } from "@rainbow-me/rainbowkit";
-import { 
+import {
   metaMaskWallet,
-  omniWallet, 
+  omniWallet,
   walletConnectWallet,
   ledgerWallet
 } from '@rainbow-me/rainbowkit/wallets';

--- a/packages/rainbowkit-celo/package.json
+++ b/packages/rainbowkit-celo/package.json
@@ -15,14 +15,6 @@
       "types": "./index.d.ts",
       "import": "./index.js"
     },
-    "./wallets": {
-      "types": "./wallets/index.d.ts",
-      "import": "./wallets/index.js"
-    },
-    "./lists": {
-      "types": "./lists/index.d.ts",
-      "import": "./lists/index.js"
-    },
     "./chains": {
       "types": "./chains/index.d.ts",
       "import": "./chains/index.js"
@@ -38,6 +30,26 @@
     "./chains/baklava": {
       "types": "./chains/baklava.d.ts",
       "import": "./chains/baklava.js"
+    },
+    "./lists": {
+      "types": "./lists/index.d.ts",
+      "import": "./lists/index.js"
+    },
+    "./wallets": {
+      "types": "./wallets/index.d.ts",
+      "import": "./wallets/index.js"
+    },
+    "./wallets/celo-terminal": {
+      "types": "./wallets/celo-terminal.d.ts",
+      "import": "./wallets/celo-terminal.js"
+    },
+    "./wallets/celo-wallet": {
+      "types": "./wallets/celo-wallet.d.ts",
+      "import": "./wallets/celo-wallet.js"
+    },
+    "./wallets/valora": {
+      "types": "./wallets/valora.d.ts",
+      "import": "./wallets/valora.js"
     }
   },
   "type": "module",

--- a/packages/rainbowkit-celo/package.json
+++ b/packages/rainbowkit-celo/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@celo/rainbowkit-celo",
   "author": "cLabs",
-  "version": "0.8.0",
+  "version": "0.8.1-beta.1",
   "description": "Quickly Setup RainbowKit for Celo",
   "homepage": "https://rainbowkit-with-celo.vercel.app/",
   "repository": {
@@ -13,6 +13,7 @@
   "exports": {
     ".": "./index.js",
     "./wallets": "./wallets/index.js",
+    "./lists": "./lists/index.js",
     "./chains": "./chains/index.js",
     "./chains/celo": "./chains/celo.js",
     "./chains/alfajores": "./chains/alfajores.js",

--- a/packages/rainbowkit-celo/package.json
+++ b/packages/rainbowkit-celo/package.json
@@ -10,6 +10,14 @@
   },
   "main": "./index.js",
   "types": "./index.d.ts",
+  "exports": {
+    ".": "./index.js",
+    "./wallets": "./wallets/index.js",
+    "./chains": "./chains/index.js",
+    "./chains/celo": "./chains/celo.js",
+    "./chains/alfajores": "./chains/alfajores.js",
+    "./chains/baklava": "./chains/baklava.js"
+  },
   "type": "module",
   "sideEffects": false,
   "license": "MIT",

--- a/packages/rainbowkit-celo/package.json
+++ b/packages/rainbowkit-celo/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@celo/rainbowkit-celo",
   "author": "cLabs",
-  "version": "0.8.1-beta.1",
+  "version": "0.8.1-beta.3",
   "description": "Quickly Setup RainbowKit for Celo",
   "homepage": "https://rainbowkit-with-celo.vercel.app/",
   "repository": {
@@ -52,6 +52,7 @@
   ],
   "scripts": {
     "prepublish": "\\cp ../../readme.md ./README.md",
+    "prepack": "yarn build",
     "lint": "eslint *.ts*",
     "clean": "tsc -b --clean",
     "build": "tsc -b"

--- a/packages/rainbowkit-celo/package.json
+++ b/packages/rainbowkit-celo/package.json
@@ -37,7 +37,7 @@
   "peerDependencies": {
     "@rainbow-me/rainbowkit": ">=0.8.1",
     "ethers": ">=5.5.1",
-    "wagmi": ">0.9.x"
+    "wagmi": "0.9.x"
   },
   "devDependencies": {
     "@types/react": "^17.0.37",

--- a/packages/rainbowkit-celo/package.json
+++ b/packages/rainbowkit-celo/package.json
@@ -11,13 +11,34 @@
   "main": "./index.js",
   "types": "./index.d.ts",
   "exports": {
-    ".": "./index.js",
-    "./wallets": "./wallets/index.js",
-    "./lists": "./lists/index.js",
-    "./chains": "./chains/index.js",
-    "./chains/celo": "./chains/celo.js",
-    "./chains/alfajores": "./chains/alfajores.js",
-    "./chains/baklava": "./chains/baklava.js"
+    ".": {
+      "types": "./index.d.ts",
+      "import": "./index.js"
+    },
+    "./wallets": {
+      "types": "./wallets/index.d.ts",
+      "import": "./wallets/index.js"
+    },
+    "./lists": {
+      "types": "./lists/index.d.ts",
+      "import": "./lists/index.js"
+    },
+    "./chains": {
+      "types": "./chains/index.d.ts",
+      "import": "./chains/index.js"
+    },
+    "./chains/celo": {
+      "types": "./chains/celo.d.ts",
+      "import": "./chains/celo.js"
+    },
+    "./chains/alfajores": {
+      "types": "./chains/alfajores.d.ts",
+      "import": "./chains/alfajores.js"
+    },
+    "./chains/baklava": {
+      "types": "./chains/baklava.d.ts",
+      "import": "./chains/baklava.js"
+    }
   },
   "type": "module",
   "sideEffects": false,

--- a/packages/rainbowkit-celo/wallets/celo-dance.ts
+++ b/packages/rainbowkit-celo/wallets/celo-dance.ts
@@ -3,7 +3,7 @@ import {
   getWalletConnectConnector,
   Wallet,
 } from "@rainbow-me/rainbowkit";
-import { Alfajores, Baklava, Celo } from "../chains";
+import { Alfajores, Baklava, Celo } from "@celo/rainbowkit-celo/chains";
 
 export interface CeloDanceOptions {
   chains: Chain[];

--- a/packages/rainbowkit-celo/wallets/celo-extension.ts
+++ b/packages/rainbowkit-celo/wallets/celo-extension.ts
@@ -1,7 +1,7 @@
 import { Chain, Wallet } from "@rainbow-me/rainbowkit";
 import { InjectedConnector } from "wagmi/connectors/injected";
 
-import { Baklava, Celo } from "../chains";
+import { Baklava, Celo } from "@celo/rainbowkit-celo/chains";
 
 export interface CeloExtensionOptions {
   chains: Chain[];

--- a/packages/rainbowkit-celo/wallets/celo-terminal.ts
+++ b/packages/rainbowkit-celo/wallets/celo-terminal.ts
@@ -3,7 +3,7 @@ import {
   getWalletConnectConnector,
   Wallet,
 } from "@rainbow-me/rainbowkit";
-import { Alfajores, Baklava, Celo } from "../chains";
+import { Alfajores, Baklava, Celo } from "@celo/rainbowkit-celo/chains";
 
 export interface CeloTerminalOptions {
   chains: Chain[];

--- a/packages/rainbowkit-celo/wallets/celo-wallet.ts
+++ b/packages/rainbowkit-celo/wallets/celo-wallet.ts
@@ -1,7 +1,7 @@
 import type { Chain, Wallet } from "@rainbow-me/rainbowkit";
 import { getWalletConnectConnector } from "@rainbow-me/rainbowkit";
 
-import { Alfajores, Baklava, Celo } from "../chains";
+import { Alfajores, Baklava, Celo } from "@celo/rainbowkit-celo/chains";
 
 export interface CeloWalletOptions {
   chains: Chain[];

--- a/packages/rainbowkit-celo/wallets/index.ts
+++ b/packages/rainbowkit-celo/wallets/index.ts
@@ -1,6 +1,5 @@
-export { default as Valora } from "./valora";
-export { default as CeloWallet } from "./celo-wallet";
-export { default as CeloTerminal } from "./celo-terminal";
-// export { default as CeloExtension } from "./celo-extension";
-export { default as CeloDance } from "./celo-dance";
+export { default as Valora } from "./valora.js";
+export { default as CeloWallet } from "./celo-wallet.js";
+export { default as CeloTerminal } from "./celo-terminal.js";
+export { default as CeloDance } from "./celo-dance.js";
 

--- a/packages/rainbowkit-celo/wallets/valora.ts
+++ b/packages/rainbowkit-celo/wallets/valora.ts
@@ -1,7 +1,7 @@
 import type { Chain, Wallet } from "@rainbow-me/rainbowkit";
 import { getWalletConnectConnector } from "@rainbow-me/rainbowkit";
 
-import { Alfajores, Baklava, Celo } from "../chains";
+import { Alfajores, Baklava, Celo } from "@celo/rainbowkit-celo/chains";
 
 // rainbowkit utils has it but doesn't export it :/
 function isAndroid(): boolean {

